### ISSUE TITLE
Change text for the New Quote Button

### ISSUE THREAD

**Issue by AndreyDmitriev39r** [AndreyDmitriev39r/random-quote-machine-v2#26]
Text `Inspire me!` appears to be relevant for inspiritional quotes, but not for all kinds of quotes.

I see two ways to go about it:

- the simple one: just come up with more generic text for button and change, it will remain static,
- the interesting one(more complicated one): associate every quote type with some text for button, and change it dynamically.

The choice is up to assignee.

**Comment by MaximusI30** [comment]
I think we should go for a dynamically changing text as well. It's gonna be so cool. 
Let's decide on the words we want to display when we have rock solid kinds of quotes.

**Comment by AndreyDmitriev39r** [comment]
> I think we should go for a dynamically changing text as well. It's gonna be so cool. Let's decide on the words we want to display when we have rock solid kinds of quotes.

agreed, then it should be blocked until the refactoring shenanigans
